code/modules/clothing/under/jobs/rnd.dm

```diff
@@ -108,9 +108,9 @@
 	name = "geneticist's jumpsuit"
 	icon_state = "genetics"
 	inhand_icon_state = "w_suit"
-	armor_type = /datum/armor/rnd_geneticist
+	armor_type = /datum/armor/clothing_under/rnd_geneticist
 
-/datum/armor/rnd_geneticist
+/datum/armor/clothing_under/rnd_geneticist
```

Comment:
oh wow look the science jobs have matching armor but they are both unique entries wow

*baby tantrum tenor gif*